terraform/modules/data-share-service/alarms.tf

```diff
@@ -0,0 +1,179 @@
+locals {
+  alarm_prefix = "${var.environment}-gdx"
+  alarms = {
+    lev_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-lev-error-rate",
```

Comment:
We should probably add the alarm actions to these, posting to the SNS, just it won't email

Reply:
Am dumb, seen this below